src/js/structs/__tests__/Service-test.js

```diff
@@ -494,6 +495,16 @@ describe('Service', function () {
 
   });
 
+  describe('#getTaskStats', function () {
+
+    it('returns task stats instance', function () {
+      let service = new Service({taskStats:{}});
+
+      expect(service.getTaskStats() instanceof TaskStats).toBeTruthy();
```

Comment:
No need to update this, but you can also `jasmine.any` to test type. 
`expect(service.getTaskStats()).toEqual(jasmine.any(TaskStats))`